@PROTOBUF_SYNTAX_VERSION@
import "dccl/protobuf/option_extensions.proto";
import "goby/test/acomms/dccl3/header.proto";

message GobyMessage1
{
  option (dccl.msg).id = 4;
  option (dccl.msg).max_bytes = 32;

  optional int32 int32_val = 1 [(dccl.field).min=0, (dccl.field).max=20];
}

message GobyMessage2
{
  option (dccl.msg).id = 5;
  option (dccl.msg).max_bytes = 32;

  optional bool bool_val = 1;
}

message GobyMessage3
{
  option (dccl.msg).id = 6;
  option (dccl.msg).max_bytes = 32;

  optional string string_val = 1 [(dccl.field).max_length=10];
}
